hybridse/include/node/node_enum.h

```diff
@@ -93,7 +93,6 @@ enum SqlNodeType {
     kDeleteStmt,
     kCreateFunctionStmt,
     kDynamicUdfFnDef,
-    kDynamicUdafFnDef,
```

Comment:
rm will break `kWithClauseEntry` with old version

Reply:
what's the impact

Reply:
value of `kWithClauseEntry` changes

Reply:
i think there is no effect of the behavior of the program although the values change. if have, please let me known